my-app/src/main/java/com/JavaWebServer/app/RESTMethods/Params.java

```diff
@@ -0,0 +1,47 @@
+package com.JavaWebServer.app;
+
+import java.util.HashMap;
+import java.util.Map;
+
+ public class Params implements RestMethod {
+  private String status;
+  private String params;
+  private Request request;
+  private Map <String, String> encodingKey;
+  private String CRLF ="\r\n";
+  private String typeHeader = "Content-Type: ";
+  private String contentLength = "Content-Length:";
+
+  public Params (String status, String params, Map <String, String> encodingKey) {
+    this.status = status;
+    this.params = params;
+    this.encodingKey= encodingKey;
+  }
+
+  public byte [] handleRequest (Request request) {
+    this.request = request;
+    String body = decodeParams();
+    int size = body.getBytes().length;
+    String response = (status+CRLF+typeHeader+"text/plain"+CRLF+contentLength+size+CRLF+CRLF+body);
```

Comment:
It seems like you are building these headers several places, with a very similar setup that doesn't seem very readable, or extensible. Can you think of a better way to build this String?